modules/fundamental/src/advisory/model/details/advisory_vulnerability.rs

```diff
@@ -283,3 +312,19 @@ impl AdvisoryVulnerabilitySummary {
         Ok(summaries)
     }
 }
+
+#[derive(Serialize, Deserialize, Debug, Clone, ToSchema)]
+pub struct NonNormativeVulnerabilityInfo {
```

Comment:
This smells a little like our old `NonLeaking...` types. Wouldn't both a normative and a non-normative info struct have the same fields?

Reply:
This is intentional. I'll document in-code.